gate-core/src/main/java/com/netflix/spinnaker/gate/services/TaskService.java

```diff
@@ -52,44 +53,45 @@ public Map create(Map body) {
       AuthenticatedRequest.setApplication(body.get("application").toString());
     }
 
-    return orcaServiceSelector.select().doOperation(body);
+    return (Map) Retrofit2SyncCall.execute(orcaServiceSelector.select().doOperation(body));
```

Comment:
I see we get an error without a cast.  What would it take to get this working without one?
```
/Users/dbyron/src/spinnaker/gate/gate-core/src/main/java/com/netflix/spinnaker/gate/services/TaskService.java:56: error: incompatible types: Object cannot be converted to Map
    return Retrofit2SyncCall.execute(orcaServiceSelector.select().doOperation(body));
```